website/pages/en/subgraphs.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Subgraphs
+---
+
+## What is a Subgraph?
+
+A custom, open API that extracts data from a blockchain, processes it, and stores it so it can be easily queried via GraphQL.
```

Comment:
Feel free to disagree, but to me it reads better as a full sentence:
```suggestion
A subgraph is a custom, open API that extracts data from a blockchain, processes it, and stores it so it can be easily queried via GraphQL.
```